views/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ps_linklist",
-  "version": "3.0.3",
+  "version": "3.0.4",
```

Comment:
In fact you don't need to set a version in your package.json since it's not a npm library :)

Reply:
Yeah I know, but since it's already here anyway ^^ You think it should be removed?
This can be done in the next version